src/__test__/bitDepth1.read.test.ts

```diff
@@ -70,14 +79,15 @@ describe('decode image with bitDepth of 1', () => {
 
   it('decode a 62x4', () => {
     const result = decode(fs.readFileSync('src/__test__/files/62x4.bmp'));
-    expect(result).toMatchSnapshot();
+    testDecode(result, '62x4.bmp');
```

Comment:
This test can be removed or changed to be like the others.

It's basically doing `expect(decode('62x4.bmp')).toEqual(decode('62x4.bmp'))`.